src/material-experimental/mdc-chips/chip.ts

```diff
@@ -373,6 +389,11 @@ export class MatChip extends _MatChipMixinBase implements AfterContentInit, Afte
     this._chipFoundation.destroy();
   }
 
+  /** Returns the directionality */
+  getDir() {
```

Comment:
just make `_dir` public instead (leave the underscore in the name)

Reply:
I got another comment saying to use a method unless it's readonly in a separate PR:
https://github.com/angular/components/pull/19985#discussion_r455796733

Perhaps the best way is to append an underscore so the method name is _getDir() ?

Reply:
Update: exposed _dir and made it readonly